Packs/MailListener_-_POP3/Integrations/MailListener_POP3/MailListener_POP3.yml

```diff
@@ -20,7 +20,7 @@ configuration:
   name: password
   required: true
   type: 4
-- defaultvalue: 'True'
+- defaultvalue: true
```

Comment:
I think that be default we set it to `false`. @Itay4 ?

Reply:
it should be `true` as we want users to use ssl by default, but this is a breaking change and should be mentioned in the release notes as such

Reply:
@bakatzir @Itay4  I don't think this is a breaking change.
I have tested it on my local server - I have configured an instance without SSL, then changed the marketplace url to use the updated url (copied it from the gitlab pipeline), then updated the pop3 pack, refreshed and the existing instance didn't change. New instance is now working as expected though (which means the change works only on new instances). Should we revert the release notes to be without the "Breaking changes"?

Reply:
Removed it.